content/modules/data-table/versions/1.7.0.um

```diff
@@ -0,0 +1,26 @@
+@version 1.5.0
+  @examples
+    @example
+      @html: @inline 1.5.0/simple-example.html
```

Comment:
I probably should have been a bit more clear :)
At the top of the file there is `@version 1.5.0` but it should be `@version 1.7.0` otherwise `quantum-version` will do the wrong thing